### PR TITLE
PR feedback

### PR DESCRIPTION
**Pull request by bonomat**
follow-up PR from https://github.com/comit-network/waves/pull/191#discussion_r677959410 and https://github.com/comit-network/waves/pull/191#discussion_r677959597

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -194,24 +194,12 @@ dependencies = [
  "hmac 0.11.0",
  "k256",
```
Comment (thomaseizinger) [inline Cargo.lock:195]: It is still here, you may need `default-features = false` on `bip32`.

Comment (thomaseizinger) [reply Cargo.lock:195]: Yep: https://github.com/iqlusioninc/crates/blob/main/bip32/Cargo.toml#L41

**Review by thomaseizinger — APPROVED** [review]
Neat!

File: Cargo.lock (hunk 1 above)
Comment (bonomat) [reply Cargo.lock]: But now it's gone :) 
[`32ba599` (#193)](https://github.com/comit-network/waves/pull/193/commits/32ba59915b96a2fb70c4c4080cd6b55e7a7164ec)

Comment (thomaseizinger) [reply Cargo.lock]: -11 dependencies :partying_face:

**Pull request merged**